chef/data_bags/crowbar/migrate/network/101_add_bonding_miimon_xmit_hash_policy.rb

```diff
@@ -15,5 +15,13 @@ def downgrade(ta, td, a, d)
   unless ta["teaming"].key? "xmit_hash_policy"
     a["teaming"].delete "xmit_hash_policy"
   end
+  a["conduit_map"].each do |conduit|
+    if a["conduit_map"][conduit].key? "team_miimon"
```

Comment:
Technically this is not needed. Deleteing a  non existing key from a hash in ruby doesn't yield an error. OTOH, it doesn't really harm either.